android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -543,6 +543,30 @@ class PlayerModule(private val context: ReactApplicationContext) : ReactContextB
         }
     }
 
+    /**
+     * Resolve `nativeId`'s current playback speed.
+     * @param nativeId Target player Id.
+     * @param promise JS promise object.
+     */
+    @ReactMethod
+    fun getPlaybackSpeed(nativeId: NativeId, promise: Promise) {
+        uiManager()?.addUIBlock {
+            promise.resolve(players[nativeId]?.playbackSpeed)
+        }
+    }
+
+    /**
+     * Call `.setPlaybackSpeed(playbackSpeed:)` on `nativeId`'s player.
```

Comment:
I think this line is not really explaining what the method does.

What do you think about this?
```suggestion
     * Sets playback speed for the player.
```

Reply:
Nice idea, I like it